impl/src/test/java/org/pentaho/commons/metadata/mqleditor/editor/service/util/MQLEditorServiceDelegateTest.java

```diff
@@ -56,7 +56,7 @@ public void convertNewThinAggregationTypeTest() {
   }
 
   // Timeout is set in order to prevent test "hanging" it's not test expected scenario
-  @Test( timeout = 5000 )
+  @Test( timeout = 10000 )
```

Comment:
Had to increase the timeout value in `org.pentaho.commons.metadata.mqleditor.editor.service.util.MQLEditorServiceDelegateTest.testConcurrency` in order for the tests to pass in `Wingman` :man_shrugging: